packages/playground/src/router/index.ts

```diff
@@ -136,6 +136,11 @@ const router = createRouter({
       component: () => import("../views/contracts_list.vue"),
       meta: { title: "Contracts List", info: { page: "info/contracts_list.md" } },
     },
+    {
+      path: "/minting",
+      component: () => import("../views/minting_view.vue"),
+      meta: { title: "Minting" },
+    },
```

Comment:
```suggestion
    {
      path: "/minting",
      component: () => import("../views/minting_view.vue"),
      meta: { title: "Minting" },
      beforeEnter(_, __, next) {
        const network = process.env.NETWORK || window.env.NETWORK;
        if (network !== NetworkEnv.dev) {
          return next({ path: "/page-not-found" });
        }
        next();
      },
    },
```

We have to add `beforeEnter` guard to avoid the user from mistakly enter /minting if he has a link

Reply:
agreed but it works on main net only